docs/sdks/ios/deeplink.md

```diff
@@ -0,0 +1,43 @@
+---
+id: Universal Link
+title: Universal Link
+---
+
+## CleverPush Universal Link
+
+1. Select the Project.
+2. Select the Project target.
+3. Select the info tab.
+4. Select the url type.
+5. Select the url scheme and enter the url scheme (Example :- cleverpush)
+6. Go to the button of any view controller from where we need to share the url to another user 
+
+```swift
+@IBAction func btnHandlerShareURL(_ sender: Any) {
+
+        // From here user can share the cleverpush url to another user using the share activity
+        let path = "cleverpush://"
+        let textToShare = [ path ]
+        let activityViewController = UIActivityViewController(activityItems: textToShare, applicationActivities: nil)
+        activityViewController.popoverPresentationController?.sourceView = self.view
+        activityViewController.excludedActivityTypes = [ UIActivity.ActivityType.airDrop, UIActivity.ActivityType.postToFacebook ]
+        self.present(activityViewController, animated: true, completion: nil)
+        
+    }
+```
+
+7. Go to AppDelegate.Swift file (We have to add the below code into the AppDelegate.file)
```

Comment:
Add the following code in your `AppDelegate.swift` file

Reply:
Resolved

Reply:
Do we have to keep the simple text or do we have to create the code block for this?